app/src/test/java/com/kickstarter/libs/PushNotificationsTest.kt

```diff
@@ -0,0 +1,29 @@
+package com.kickstarter.libs
+
+import android.app.Application
+import androidx.test.core.app.ApplicationProvider
+import com.kickstarter.KSRobolectricTestCase
+import com.kickstarter.mock.factories.MessageThreadEnvelopeFactory
+import com.kickstarter.mock.factories.PushNotificationEnvelopeFactory
+import com.kickstarter.ui.IntentKey
+import org.junit.Test
+
+class PushNotificationsTest : KSRobolectricTestCase() {
+    private val application: Application = ApplicationProvider.getApplicationContext()
```

Comment:
`KSRobolectricTestCase` has a protected method `application()` to fetch the context Application, take a look at `SegmentTest.kt:73`

Reply:
👍🏻